hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -102,27 +124,237 @@
 	private int decimals;
 	private long autoRenewPeriod;
 	private long lastUsedSerialNumber;
+	static Predicate<Key> REMOVES_ADMIN_KEY = ImmutableKeyUtils::signalsKeyRemoval;
+	private JKey feeScheduleKey;
```

Comment:
Let's be consistend and place this key near the other key definitions ( rows 111 -  113 )